recipes-core/rauc/rauc-conf.bb

```
@@ -0,0 +1,31 @@
+# Copyright (C) 2020 Enrico Jorns <ejo@pengutronix.de>
+# Released under the MIT license (see COPYING.MIT for the terms)
+
+SUMMARY = "RAUC system configuration"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COREBASE}/meta/COPYING.MIT;md5=3da9cfbcb788c80a0384361b4de20420"
+
+RAUC_KEYRING_FILE ??= "ca.cert.pem"
+RAUC_KEYRING_URI ??= "file://${RAUC_KEYRING_FILE}"
+
+SRC_URI:append = " \
+  file://system.conf \
+  ${RAUC_KEYRING_URI} \
+  "
+
+do_install () {
+        # Create rauc config dir
+        # Warn if system configuration was not overwritten
+        if ! grep -q "^[^#]" ${WORKDIR}/system.conf; then
+                bbwarn "Please overwrite example system.conf with a project specific one!"
+        fi
+        install -d ${D}${sysconfdir}/rauc
+        install -m 0644 ${WORKDIR}/system.conf ${D}${sysconfdir}/rauc/system.conf
```

Perhaps simplify in accordance to the installation of the keyring file:
```suggestion
        install -m 0644 ${WORKDIR}/system.conf ${D}${sysconfdir}/rauc/
```

---

I kept it the same at first, making as few changes as necessary compared to the original recipe.